### PR TITLE
KAFKA-17685 make sure all modules have consistent testRuntimeOnly dependencies

### PR DESCRIPTION
**Pull request by m1a2st**
Jira: https://issues.apache.org/jira/browse/KAFKA-17685

we declare a variable of runtimeLibs to simplfied the testRuntimeOnly dependency

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -122,6 +122,11 @@ ext {
     if (name == "compileTestJava" || name == "compileTestScala")
       options.compilerArgs << "-parameters"
   }
+
+  runtimeLibs = [
+          libs.slf4jReload4j,
+          libs.junitPlatformLanucher,
```
Comment (mumrah) [inline build.gradle]: nit: indentation is off

**Review by mumrah — APPROVED** [review]
LGTM, thanks @m1a2st! Left one minor style nitpick.

File: build.gradle [hunk 2]
```diff
@@ -122,6 +122,11 @@ ext {
     if (name == "compileTestJava" || name == "compileTestScala")
       options.compilerArgs << "-parameters"
   }
+
+  runtimeLibs = [
```
Comment (mumrah) [inline build.gradle]: Should also call this "runtimeTestLibs" probably to avoid confusion.

Comment (m1a2st) [reply build.gradle]: Thanks, It make sense to me

**Pull request merged**